.github/ISSUE_TEMPLATE/Emulation_bug_report.yaml

```diff
@@ -0,0 +1,76 @@
+# Docs - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug Report
+description: Report a issue with the emulation (ie. graphical issue, crashes, unintended behavior with doesn't occur on hardware).
+title: "[BUG]: "
```

Comment:
Is having `[BUG]: ` in the title of every bug necessary?

Reply:
probably not, just thought it might be easy to separate issues at glance. reverted.

Reply:
@Triticum0 Thanks. Sorry, but you might have to click [*Resolve conversation*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#resolving-conversations) on each review that you've completed. Apparently the reviewer can't do it unless they have write access to the repo :confused:

To save us some additional button clicking, there are 2 remaining typos after https://github.com/cemu-project/Cemu/pull/47/commits/f8721b3d04c418491cac170b03a7191811f81e7e:

- `decription` -> `description`
- `reprodue` -> `reproduce`

Reply:
Fixed the typos